source/objects/bodyClass.m

```diff
@@ -183,6 +183,21 @@ function loadHydroData(obj, hydroData)
             obj.dof_gbm   = obj.dof-6;
         end
         
+        function dragForcePre(obj,rho)
```

Comment:
@dforbush2 This implementation allows for a non-hydrodynamic body to include a linear and a quadratic viscous drag forcing term; however, based on the conversation on issue #384 the requester is asking for the Morison Element forcing which includes a drag and added inertia coefficient that also includes the flow velocity and acceleration of the undisturbed wave velocity. Therefore, this modification may not be fully addressing the issue; however, it should be relatively straightforward to adapt for the Morison Element.

Reply:
@dforbush2 My apologies, believe I misunderstood the implementation. Please disregard my previous comment.

Reply:
I did not need to modify anything to specifically incorporate the morison elements, but I did test the new implementation with morison elements on/off and they seem to be behaving correctly.

Reply:
@dforbush2 Yes, I realized that the linear and quadratic damping are attached to the body.hydroforce variable, while all of the Morison Element coefficients stay within the body object and is not placed within body.hydroforce. Perhaps the Morison Element definition should also be moved into body.hydroforce to be consistent with the rest of the hydrodynamic forces.